source/includes/fact-data-prep-oracle-step1.rst

```diff
@@ -4,18 +4,23 @@ instance. Alternatively, you can use an existing Oracle
 service account to connect to Relational Migrator with 
 the appropriate permissions.
 
+.. tip::
+
+   To migrate data from a multi-tenant container database, use a `Common
+   User
```

Comment:
Not sure how deep we want to go into this, but there are more steps before creating a common user that aren't obvious- [creating tablespaces](https://oracle-base.com/articles/12c/multitenant-manage-users-and-privileges-for-cdb-and-pdb-12cr1>) in the CDB and the PDB's.

Reply:
The referenced article mentions that "The DEFAULT TABLESPACE, TEMPORARY TABLESPACE, QUOTA and PROFILE must all reference objects that exist in all containers," so how exhaustively do we want to cover the concept on our side? Documenting other products is a slippery slope.

At minimum, I updated the link text to "create tablespaces and a Common User"

Reply:
I'm happy with that, thanks Nick. Agreed that we shouldn't describe every step in detail, especially since we expect that users will view other documentation in conjunction with ours.